docs/readmes/orc8r/upgrade_1_3.md

```diff
@@ -0,0 +1,92 @@
+---
+id: upgrade_1_3
+title: Upgrade to v1.3
+hide_title: true
+---
+
+# Upgrade to v1.3
+
+This guide covers upgrading Orchestrator deployments from v1.2 to v1.3.
+Upgrades which skip versions (e.g. v1.1 to v1.3) are not explicitly documented
```

Comment:
Move this disclaimer to an intro page for the upgrades sub-section -- either that or add it as a disclaimer at the top of all the upgrade pages